make-deb.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Make a Boulder Debian package at $PWD.
```

Comment:
This should indicate that it installs some packages, and that it expects to run on Ubuntu.